README-dev.md

```diff
@@ -220,6 +226,8 @@ https://storage.googleapis.com/mina-seed-lists/mainnet_seeds.txt.
 The `daemon.json` config file also contains bootstrap data that is specific to the network the node is trying to connect to and must be tailored specifically for a particular network. This file can also override some of the configuration options selected during compilation. The `daemon.json` file can be extracted from the Docker image
 that is dedicated to running a particular network. If it's not located in the `config` directory, it can be pointed to with `--config-file` option.
 
+The aforementioned bootstrap data includes the genesis ledger, i.e. the initial state of the blockchain. It is crucial for all the nodes on the network to have the same genesis ledger. While starting a new network, it is important that it contains at least one account possessing some funds. Otherwise, the network will not be able to bootstrap, as there will be no way to determine the next block producer.
```

Comment:
```suggestion
The previously mentioned bootstrap data includes the genesis ledger, which is the initial state of the blockchain. All the nodes on the network must have the same genesis ledger. A new network must contain at least one account with some funds before it can be started. If the network cannot determine the next block producer, it will not be able to bootstrap. 
```

Reply:
r/w for clarity